<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.arl.fjage.test.fjagecTest" tests="1" skipped="0" failures="1" errors="0" timestamp="2024-04-10T18:24:57" hostname="sweta-VM" time="7.098">
  <properties/>
  <testcase name="fjageCTest" classname="org.arl.fjage.test.fjagecTest" time="7.098">
    <failure message="java.lang.AssertionError: expected:&lt;2&gt; but was:&lt;0&gt;" type="java.lang.AssertionError">java.lang.AssertionError: expected:&lt;2&gt; but was:&lt;0&gt;
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:101)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite$StaticMetaMethodSiteNoUnwrap.invoke(StaticMetaMethodSite.java:131)
	at org.codehaus.groovy.runtime.callsite.StaticMetaMethodSite.callStatic(StaticMetaMethodSite.java:100)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallStatic(CallSiteArray.java:55)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:196)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callStatic(AbstractCallSite.java:216)
	at org.arl.fjage.test.fjagecTest.fjageCTest(fjagecTest.groovy:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.runTestClass(JUnitTestClassExecutor.java:110)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:58)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecutor.execute(JUnitTestClassExecutor.java:38)
	at org.gradle.api.internal.tasks.testing.junit.AbstractJUnitTestClassProcessor.processTestClass(AbstractJUnitTestClassProcessor.java:62)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$2.run(TestWorker.java:176)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:133)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[Running automated tests.
C : out = make: Entering directory '/home/sweta/Documents/fjage/gateways/c'
rm -fr libfjage.a libfjage.dylib libfjage.so test_fjage build
cmake -DCMAKE_C_FLAGS="" -S . -B build -DBUILD_TESTING=ON
make: Leaving directory '/home/sweta/Documents/fjage/gateways/c'
 
 err = /bin/sh: 1: cmake: not found
make: *** [Makefile:11: test] Error 127
 
 ret = 2
]]></system-out>
  <system-err><![CDATA[1712773498865|INFO|org.arl.fjage.remote.MasterContainer@1:openTcpServer|Listening on port 5081
1712773498876|INFO|org.arl.fjage.connectors.TcpServer@275:run|Listening on port 5081
1712773499803|INFO|org.arl.fjage.connectors.WebServer@1:add|Adding web context: /
1712773499868|INFO|org.arl.fjage.connectors.WebServer@1:add|Adding web context: /shell/ws
1712773500320|WARNING|org.arl.fjage.connectors.WebServer@1:start|java.io.IOException: Failed to bind to /127.0.0.1:8080
1712773503740|WARNING|org.jline@1:logr|The Parser of class org.arl.fjage.shell.ConsoleShell$1 does not support the CompletingParsedLine interface. Completion with escaped or quoted words won't work correctly.
1712773504060|INFO|org.arl.fjage.connectors.WebServer@1:add|Adding web context: /ws
1712773504071|WARNING|org.arl.fjage.connectors.WebServer@1:start|java.io.IOException: Failed to bind to /127.0.0.1:8080
1712773504076|INFO|org.arl.fjage.remote.MasterContainer@1:addConnector|Listening on ws://127.0.1.1:8080/ws
1712773504110|INFO|org.arl.fjage.remote.MasterContainer@1:init|Initializing agents...
1712773504121|INFO|org.arl.fjage.shell.ShellAgent@289:init|Agent shell init
1712773504222|INFO|org.arl.fjage.remote.MasterContainer@1:init|Agents ready...
1712773504327|INFO|org.arl.fjage.remote.MasterContainer@1:start|Starting container...
1712773504617|INFO|org.arl.fjage.connectors.WebServer@1:remove|Removing web context: /ws
1712773504630|INFO|org.arl.fjage.remote.MasterContainer@1:connectionClosed|Connection ws://127.0.1.1:8080/ws closed
1712773504631|INFO|org.arl.fjage.remote.MasterContainer@1:shutdown|Initiating shutdown...
1712773504631|INFO|org.arl.fjage.connectors.TcpServer@275:run|Stopped listening
1712773504634|INFO|org.arl.fjage.shell.ShellAgent@289:shutdown|Agent shell shutdown
1712773504644|INFO|org.arl.fjage.connectors.WebServer@289:remove|Removing web context: /shell/ws
1712773504646|INFO|org.arl.fjage.connectors.WebServer@289:stop|Stopping web server
1712773504668|INFO|org.arl.fjage.remote.MasterContainer@295:shutdown|Initiating shutdown...
1712773504691|INFO|org.arl.fjage.remote.MasterContainer@1:shutdown|All agents have shutdown
]]></system-err>
</testsuite>
